sklearn/utils/estimator_checks.py

```diff
@@ -113,12 +114,12 @@ def _yield_classifier_checks(name, Classifier):
     # basic consistency testing
     yield check_classifiers_train
     yield check_classifiers_regression_target
-    if (name not in ["MultinomialNB", "LabelPropagation", "LabelSpreading"]
+    if name not in ["MultinomialNB", "LabelPropagation", "LabelSpreading"]:
         # TODO some complication with -1 label
-            and name not in ["DecisionTreeClassifier",
-                             "ExtraTreeClassifier"]):
+        if name not in ["DecisionTreeClassifier", "ExtraTreeClassifier"]:
```

Comment:
why did you change this from `and` to a separate `if`. That's what creates the errors in your screenshot.